### ISSUE TITLE
MPI error

### ISSUE THREAD

**Issue by sarti-luca** [KratosMultiphysics/Kratos#1564]
Dear All,

I'm trying to use the MPI parallelization to solve a fluid dynamic problem. 
As first attempt I used the 3D benchmark "Cylinder in air flow" yet set up in GiD, with a coarse mesh. 
I get an error that starts with the following lines:

Traceback (most recent call last):
  File "MainKratos.py", line 13, in <module>
Traceback (most recent call last):
  File "MainKratos.py", line 13, in <module>
Traceback (most recent call last):
  File "MainKratos.py", line 13, in <module>
Traceback (most recent call last):
  File "MainKratos.py", line 13, in <module>

I attach the complete job.err file and the folder with the problem if can be useful.
Thanks for any help.

Luca

[job.err.tar.gz](https://github.com/KratosMultiphysics/Kratos/files/1749760/job.err.tar.gz)
[prova_MPI.gid.tar.gz](https://github.com/KratosMultiphysics/Kratos/files/1749734/prova_MPI.gid.tar.gz)

**Comment by RiccardoRossi** [comment]
just to be sure, before we spend time on this: can u confirm that the file is created with the latest version of the problemtype, as downloaded from:

https://github.com/KratosMultiphysics/GiDInterface

?

**Comment by sarti-luca** [comment]
@RiccardoRossi thanks for the answer. 
The file is created with the 5.2 version downloaded from the GiD repository, so is not the latest.
I'll try with that version.

Thanks.

**Comment by RiccardoRossi** [comment]
I think there have been some modifications, so i believe it is not compatible. @rubenzorrilla , @jcotela can u confirm?

**Comment by sarti-luca** [comment]
@RiccardoRossi 
I downloaded the latest version 5.3.0 both on desktop pc (right side of image below) and on cluster (left side) where I tried the MPI parallelization.

![image](https://user-images.githubusercontent.com/35461057/36796009-c2f2e2d0-1ca4-11e8-8fec-c08671d66fae.png)

I created a new problem type.
The openmp works on desktop while I get this error on cluster:

Traceback (most recent call last):
  File "MainKratos.py", line 13, in <module>
    project_parameters = Parameters( parameter_file.read())
Boost.Python.ArgumentError: Python argument types in
    Parameters.__init__(Parameters, str)
did not match C++ signature:
    __init__(P7_object, N6Kratos10ParametersE)
    __init__(P7_object, NSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE)

Instead with MPI I get the same error as in the first post.
What could be the error?
Thanks

Luca

I attach the two zip files
[ProvaNuovoKratos_MPI.gid.tar.gz](https://github.com/KratosMultiphysics/Kratos/files/1767779/ProvaNuovoKratos_MPI.gid.tar.gz)
[ProvaNuovoKratos_OPENMP.gid.tar.gz](https://github.com/KratosMultiphysics/Kratos/files/1767780/ProvaNuovoKratos_OPENMP.gid.tar.gz)
